python/pylibcudf/pylibcudf/transform.pyx

```diff
@@ -129,16 +131,19 @@ cpdef Column mask_to_bools(Py_ssize_t bitmask, int begin_bit, int end_bit):
     return Column.from_libcudf(move(c_result))
 
 
-cpdef Column transform(Column input, str unary_udf, DataType output_type, bool is_ptx):
-    """Create a new column by applying a unary function against every
-       element of an input column.
+cpdef Column transform(list[Column] inputs,
+                       str transform_udf,
+                       DataType output_type,
+                       bool is_ptx):
+    """Create a new column by applying a transform function against
+       multiple input columns.
```

Comment:
I see there are no tests for this interface. 
I wonder if it there is a plan for Python to use this and whether it is not worth keeping.

Reply:
Yes, I couldn't find one as well, I was informed it was an unused code path.
I had the impression it was a Python-first interface as the UDF transform function is made to match NUMBA's function signature. I'd have to consult with @vyasr or other Python code owners.

Reply:
I've added the transforms tests for pylibcudf